src/pk-main.c

```diff
@@ -196,7 +196,7 @@ main (int argc, char *argv[])
 	if (backend_name == NULL || g_strcmp0 (backend_name, "auto") == 0) {
 		ret  = pk_util_set_auto_backend (conf, &error);
 		if (!ret) {
-			g_printerr ("Failed to resolve auto: %s\n", error->message);
+			g_printerr (_("Failed to resolve auto: %s\n"), error->message);
```

Comment:
same:

    g_printerr ("%s: %s\n", _("Failed to resolve auto"), error->message);

and TRANSLATOR comment please.

Reply:
Can’t use a format string `%s: %s` as the translators might want to quote the error message or add text after it.

I’ve used an intermediate variable

Reply:
I don't think we've done that in other places tho, right?

Reply:
No reason to get it wrong here too